glossary.html

```diff
@@ -19,15 +19,31 @@
 <header id="top">
     <nav>
         <ul>
-            <p><a herf=index.html>Home</a></p>
-            <p><a herf=links.html>Links</a></p>
-            <p><a herf=glossary.html>glossary</a></p>
-            </ul>
-            </nav> </header>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="links.html">Links</a></li>
+            <li><a href="glossary.html">Glossary</a></li>
+        </ul>
+    </nav>
+</header>
+<main> 
+    <h1>Glossary</h1>
+    <dl>
+        <dt>HTML</dt><dd>HyperText Markup Language</dd>
+        <dt>structure</dt><dd>layout of the web document</dd>
+        <dt>Block-level elements</dt><dd>form a visble block and appear on there own line </dd>
+        <dt>Doctype</dt> <dd>tells browser how to interpret html</dd>
+        <dt>CSS</dt> <dd>means Cascading Style sheets </dd>
+    </dl>
+</main>
+    
+<footer>
+    <hr> 
+    <blockquote> The way I see it if you want the rainbow you gotta put up with the rain 
+        <cite>&mdash; <span class="author">Dolly Parton</span></cite>
+    </blockquote>  
+    <p>&copy; 2020 <a href="mailto:20023467@saultcollege.ca?subject=Message%20from%20CSD120%20website"> Matthew Burgan </a></p>
+</footer>
+<a href="#top" title="Back to top">◤</a>
 </div>
-<main> <h1> glossary
-    <li> HTML means HyperText Markup Language</li>
-    <li> structure (layout of the web document) </li>
-    <li> Block-level elements (form a visble block and appear on there own line) </li>
-    <li> Doctype tells browser how to interpret html </li>
-    <li> CSS means Cascading Style sheets </li>
```

Comment:
A definition list consists of a `<dl>` element that contains pairs of `<dt>` and `<dd>` elements